licenses/more-licences.md

```diff
@@ -1,7 +1,7 @@
 
 # agent
 ## Dependency License Report
-
+_2024-04-24 14:54:09 CEST_
```

Comment:
We wanted to generate with no date so that the output was idempotent. Can be fixed later

Reply:
That's a very good catch, I forgot to run `updateLicensesAndNotice` which does exactly that and only run `generateLicenseReport`. I've fixed it in the last commit.